test/integration/tests/funding/venmo/index.js

```diff
@@ -77,13 +77,13 @@ describe(`venmo desktop web button test `, () => {
     });
 
 
-    it(`should display button when it's passed in as instrument on desktop web`, (done) => {
+    it(`should not display button when it's passed in as instrument on desktop web`, (done) => {
```

Comment:
Ideally we would have a test for each scenario (displaying and not displaying). What do you think about using the localstorage opt-in for the tests?

For example, we can use `localStorage.setItem('enable_venmo_desktop', true)` to opt into Venmo Desktop. And we could set this to false for the negative test case. That way the ramping percentage does not impact the test cases.

Reply:
ah ignore this. Just realized that this file is no longer changed.